README.md

```diff
@@ -4,10 +4,21 @@
 
 We (La Roue Verte) are sharing this library we've built during years to ease handling java dates. We share it with the community.
 
-This code is licensed in Gnu GPL v3
+This code is licensed in Gnu GPL v3.
 
 All comments and suggestions are welcomed !
 
+## Installation
```

Comment:
There was missing a short installation note since it's now published on maven central

Reply:
Thanks!